### PR TITLE
Additional UX fixes

### PR DESCRIPTION
**Pull request by seanyesmunt**
#### Changes
Some changes based on comments [here](https://github.com/lbryio/lbry-app/pull/1650#issuecomment-398911888)
- Brought back the sticky header. Based on @Invariant-Change's design
- Send tip icon is now a present
- Change the download icon color, purple looked on light mode.
  - Green looks better, but doesn't stand out as much as black. Maybe we should just use black on light mode?
- Slightly improve spacing on file page (you are right, there are some kinks to be worked out with form row/form elements that are causing inconsistent spacing)
  - Will revist, but better than what we currently have

#### Images
screen shot 2018-06-25 at 2 21 11 am
screen shot 2018-06-25 at 2 24 16 am
screen shot 2018-06-25 at 2 26 30 am
screen shot 2018-06-25 at 2 48 53 am

screen shot 2018-06-25 at 2 08 15 am
screen shot 2018-06-25 at 2 08 46 am

### CODE REVIEW

File: src/renderer/scss/component/_header.scss [hunk 1]
```diff
@@ -1,17 +1,37 @@
 .header {
+  position: fixed;
```
Comment (kauffj) [inline src/renderer/scss/component/_header.scss]: Now that the header is fixed, should the address bar stretch to take the remaining space available?

File: src/renderer/scss/component/_header.scss [hunk 2]
```diff
@@ -1,17 +1,37 @@
...
+.header__navigation {
+  display: flex;
+  justify-content: space-between;
+}
+
+.header__history {
+  display: flex;
+  padding: 0 $spacing-width * 1/2;
+
+  @media only screen and (min-width: $medium-breakpoint) {
```
Comment (kauffj) [inline src/renderer/scss/component/_header.scss]: Is there a reason we set rules to only screen?

File: src/renderer/scss/component/_header.scss (hunk 1 above)
Comment (seanyesmunt) [reply src/renderer/scss/component/_header.scss]: yes

File: src/renderer/scss/component/_header.scss (hunk 2 above)
Comment (seanyesmunt) [reply src/renderer/scss/component/_header.scss]: I guess not since screen will be the only one used.

Comment (Invariant-Change) [reply src/renderer/scss/component/_header.scss]: `Media Types` are **optional** and if **not** used it defaults to `All`. Unless targeting a **printer** or **screen reader**, there is no need for them.

**Pull request merged**